Formula/csvq.rb

```diff
@@ -0,0 +1,17 @@
+class Csvq < Formula
+  desc "SQL-like query language for csv"
+  homepage "https://mithrandie.github.io/csvq"
+  url "https://github.com/mithrandie/csvq/archive/v1.11.1.tar.gz"
+  sha256 "e6423098bc9ef4bd54d482331a3a8a88dc90b15ed2978ba7860fffba46765c68"
+
+  depends_on "go" => :build
+
+  def install
+    system "make"
+    bin.install "csvq"
+  end
+
+  test do
+    system "#{bin}/csvq", "--version"
```

Comment:
Insufficient test. Please run an actual query on an actual csv file, and verify output.

Thanks!

Reply:
Thank you for your comment. I already added test for actual query.